cli-feed-v4.json

```diff
@@ -97,11 +98,12 @@
               "displayVersion": "v2",
               "targetFramework": ".NET Core"
             },
-            "capabilites": "",
+            "capabilities": "",
             "sdk": {
               "name": "Microsoft.NET.Sdk.Functions",
               "version": "1.0.31"
             },
+            "default": true,
```

Comment:
@ejizba, I added `default` properties to the `dotnet` under `workerRuntimes`. Because it's an object rather than an array, I figured the order may not necessarily persist. So I added this to make it clearer.

Reply:
Looks good. The order should persist for us just fine regardless of array/object